etc/gensbom/gensbom.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euo pipefail
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+
+WORKSPACE=${WORKSPACE:-${HERE}/.gensbom}
+HTTP_COMMAND=${HTTP_COMMAND:-http}
+SYFT_COMMAND=${SYFT_COMMAND:-syft}
```

Comment:
We're not doing the users any favors by assuming they have our dependencies installed. I think we should create an appropriate `Dockerfile` that contains the script and its dependencies to be invoked using `docker run ...` or some such.

Reply:
Done.